README.md

```diff
@@ -49,6 +49,48 @@ If you'd like to protect your Sentry install with SSL/TLS, there are
 fantastic SSL/TLS proxies like [HAProxy](http://www.haproxy.org/)
```

Comment:
Heh, we should definitely remove this line as it predates us adding Nginx to the mix.

Reply:
done